scripts/release-update-tilt-docs-repo.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Updates the Tilt docs repo with the latest version info.
+#
+# Usage:
+# scripts/update-docs-tilt-repo.sh $VERSION
+# where VERSION is of the form 0.1.0
+
+set -e
+
+if [[ "$GITHUB_TOKEN" == "" ]]; then
```

Comment:
```suggestion
if [[ "${GITHUB_TOKEN-}" == "" ]]; then
```

(needed if you accept the `set -u`)

Reply:
done